### ISSUE TITLE
Code refactor and improvement

### ISSUE THREAD

**Issue by CristhianRodriguezMolina** [kommitters/mintacoin#55]
### Describe the solution you'd like

We need to do a refactor in order to improve the code and remove some unnecessary statements and validations.

### Describe alternatives you've considered

- [ ] Remove errors from the controller and put them in the fallback, and leave a default error if it's an unknown code error. 
- [ ] Move the error rendering to the error view.
- [ ] Add an env variable for the starting balance when creating an account.
- [ ] Remove the UUID validation with regex from the schemas.
- [ ] Improve the functional programming of the stellar implementations.
- [ ] Update the schemas types from `asset_id: Asset.t(), wallet_id: wallet.t()` to `asset: Asset.t(), wallet: Wallet.t()`
- [ ] Unify the crypto folders.

**Comment by samu975** [comment]
Hola, buenas tardes. ¿Puedo trabajar en este issue?

**Comment by SanGeeky** [comment]
@samu975 Of course, go ahead, please review the issues in the list with the label ['Good First Issue'](https://github.com/kommitters/mintacoin/labels/%F0%9F%91%8B%20Good%20first%20issue), and write into the Issue which wants to contribute to be possible assigned it to you.
